### DISCUSSION TITLE
[Feature Request] Seq for MacOS

### DISCUSSION THREAD

**Discussion by ChaseFlorell** [datalust/seq-tickets#1249] [Ideas]
We're building a rather large enterprise iOS application in Xamarin Forms. We're looking to turn on serilog structured logging, but currently don't have a tool to read / view / parse / search those structured logs on MacOS.

Are there any plans to migrate seq over to a cross platform offering so that it can be used on operating systems other than Windows?

**Comment by nblumhardt** [comment]
Hi Chase, thanks for dropping by! You can use Seq on macOS today via the Docker container:

https://docs.datalust.co/docs/getting-started-with-docker

A native macOS build isn't out of the question in future, but Docker does a nice job of it currently. Does this help?

Best regards,
Nick

**Reply by ChaseFlorell** [reply]
I can absolutely take a run at using Docker, but it's not feasible for our entire development/testing team of 150 people.
I'll give it a run personally.

**Reply by nblumhardt** [reply]
Thanks for your reply! Keen to hear how you go 👍 

So that we can understand the space better - do you mean that installing Docker itself would be a challenge in your environment, or running a command-line to get the container started?

If installing Docker is the problematic bit, what would the important differences provided by a macOS Seq installer?

Thanks again!

**Reply by nblumhardt** [reply]
(Also - assuming you have ruled out running a central Seq instance for your dev team? Individual API keys with attached properties can effectively separate the logs from each user.)
